docs/catalog/routes.js

```diff
@@ -64,79 +64,34 @@ export default [
     content: pageLoader(() => import('../../README.md'))
   },
   {
-    title: 'Foundation',
+    title: 'Visual Style',
```

Comment:
My only quibble with this PR is that I question renaming this section from "Foundation" to "Visual Style". 

I just did a quick survey of some other Guides and I see the following language used for the section that holds Typogra[phy, Colours, etc:

## Foundation
https://atlassian.design/guidelines/product/foundations/color
http://www.bbc.co.uk/gel/guidelines/category/foundations
https://material.io/design/introduction/

## Style
http://www.carbondesignsystem.com/style/color/swatches
https://design-system.service.gov.uk/styles/
https://docs.microsoft.com/en-us/windows/uwp/design/style/index

## Design
http://rizzo.lonelyplanet.com/styleguide/design-elements/colours
https://marvelapp.com/styleguide/overview/introduction

## Visual Style
https://comet.discoveryeducation.com

## Guidelines
https://www.lightningdesignsystem.com/guidelines/overview/

## Support
https://styleguide.github.com/primer/support/color-system/

Reply:
Not a super blocker ... I just always prefer one word names to two-word names. 

At any rate, I'm approving the PR with this comment standing. @matthewlyle you can do what you want with it.

Reply:
Yeah I mostly went with Visual Style as I had been reading Nathan Curtis' documentation articles and he calls that section "Visual Style" or "Visual Language" in all of his design systems. I prefer one word too. How do you feel about Style?